runtime/stateManager.py

```diff
@@ -288,9 +289,11 @@ def hibike_response_device_disconnect(self, uid):
         del devs[uid]
 
     def hibike_response_timestamp_up(self, *data):
+        print("We have received timestamp_up")
```

Comment:
Might be a good idea to remove this, since we already know it's working when dawn receives a timestamp back.